### PR TITLE
Add index service to index a single ead

### PR DESCRIPTION
**Pull request by hackmastera**
Used in development to get a bit of data in more quickly. Not sure
whether this is the desired architecture.

### CODE REVIEW

**Comment by hackmastera** [comment]
Looks like this was extracted from the Rakefile. If the test can be made to pass we should replace the contents of that rake task with this service.

File: app/services/index_service.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class IndexService
+  # Index an EAD-XML Document into Solr
+  # @param [String] relative_path
+  def index_document(relative_path:, root_path: nil)
+    root_path ||= pulfa_root
```
Comment (jrgriffiniii) [inline app/services/index_service.rb]: Is `pulfa_root` `nil` here?

**Comment by hackmastera** [comment]
I believe progress here is blocked by #183 and the solr cleaning mentioned therein.

**Comment by jrgriffiniii** [comment]
Deprecated by #256

**Pull request closed without merging**